assets/xml/code/fbdemo_wipe1.xml

```diff
@@ -1,10 +1,10 @@
 <Root>
-    <!-- ZAPD isn't finding sWipe1Vtx and sWipe1Tex properly for sWipe1DL, instead finding symbols much later in `code`-->
-    <File Name="code" OutName="z_fbdemo_wipe1" RangeStart="0x10E2A0" RangeEnd="0x10EC30" Segment="0">
-        <Array Name="sWipe1Vtx" Count="25" Offset="0x10E2A0">
+    <File Name="code" OutName="z_fbdemo_wipe1" RangeStart="0x10E2A0" RangeEnd="0x10EC30">
+        <Array Name="sTransWipeVtx" Count="25" Offset="0x10E2A0">
             <Vtx/>
         </Array>
-        <Texture Name="sWipe1Tex" Format="i4" Width="64" Height="64" Offset="0x10E430"/>
-       <!-- <DList Name="sWipe1DL" Offset="0x10EC30"/>-->
+        <Texture Name="sTransWipeTex" Format="i4" Width="64" Height="64" Offset="0x10E430"/>
+        <!-- ZAPD isn't finding symbols correctly, instead finding them much later in `code`-->
+        <!-- <DList Name="sTransWipeDL" Offset="0x10EC30"/> -->
```

Comment:
Would you know if this explanation is why this texture doesn't have an `OutName="wipe_tex"`?

Reply:
Nah, I think it was just forgotten as it works fine.